pyslicer/utils/validators.py

```diff
@@ -200,12 +210,25 @@ def _has_empty_field(self):
                     "The value for an id should be a dictionary")
         return False
 
+    def check_indexation_size(self):
+        indexation_size = len(self.data)
+
+        # auto-create-fields property should not be considered as indexation
+        if "auto-create-fields" in self.data:
+            indexation_size -= 1
+
+        if indexation_size > 1000:
+            raise exceptions.InvalidIndexException(
+                "Your index command shouldn't have more than 1000 values.")
```

Comment:
1000 is hard coded

refactor maybe?

use global constants or maybe config file.

probably global constant as we don't want customers playing around with this